.babelrc

```diff
@@ -1,7 +1,6 @@
 {
   "presets": [
-    "latest",
-    "react"
+    "react-native"
```

Comment:
It builds just fine and tests pass. Dev also works. I thinks it just a matter of picking out which transforms are missing in the react + latest preset: https://www.diffchecker.com/Sez4D019

I know that babel-plugin-transform-class-properties is needed for sure, after applying that the next error that appeared didn't really specify what else was missing: `babelhelpers.typeof is not a function`